src/r502_commands.c

```diff
@@ -471,6 +480,48 @@ static int32_t store_pkg(uint8_t *pkg, Command command, int32_t pkg_len) {
     return SUCCESS;
 }
 
+
+static int32_t aura_led_config(uint8_t *pkg, Command command, int32_t pkg_len) {
```

Comment:
Keep the same order of the commands as in the `cmd_func` array.

Reply:
Done!